main.go

```diff
@@ -454,11 +455,33 @@ func (cacheModel *CacheModel) CompareFilePathMaps(currentFilePathsMap map[string
 	return triggerNewCache, nil
 }
 
+func cleanDuplicatePaths(paths []string) []string {
+	cleanedPaths := []string{}
+	for _, item := range paths {
+		if item == "" {
+			continue
+		}
+		cleanPath := path.Clean(item)
+		cleanedPaths = append(cleanedPaths, cleanPath)
```

Comment:
Please use: https://github.com/bitrise-io/go-utils/blob/master/sliceutil/sliceutil.go#L30 for checking if `cleanPath` is already in the `cleanedPaths` slice. This way the following lines is not required.

Reply:
done